recipes/fermikit/test-data/test.fastq

```diff
@@ -0,0 +1,100 @@
+@phiX174_29_494_0_1_0_0_2:0:0_3:0:0_0
```

Comment:
> We can simply not effort functional tests. Space limitations and so on come to my mind.

I agree for the space limitations, but this is just 100 lines here.

> bioperl is known to be broken and there are several issues

it used to work ... 1.6.924=pl5.22.0_0 still works, while newer packages are broken. Of course having tests wouldn't make it easier to upgrade the recipe, but at least you had some level of confidence that installing bioperl will give you a halfway functional package.

> One thing you could do without populating the github repo is to move this into the build script, fetch the fasta etc ... not ideal, but better than nothing.

In principle I like this, but it wouldn't test the runtime dependency. What would be cool is if you could stick urls in the `files:` section.